Example/Tests/WordPressXMLRPCApiTests.m

```diff
@@ -18,12 +18,40 @@ - (void)tearDown {
 }
 
 - (void)testGuessXMLRPCURLForSite {
-    XCTestExpectation *expectation = [self expectationWithDescription:@"Call should fail with erro when invoking with empty string"];
+    __block NSError *errorToCheck = nil;
+    XCTestExpectation *expectationEmpty = [self expectationWithDescription:@"Call should fail with error when invoking with empty string"];
     [WordPressXMLRPCApi guessXMLRPCURLForSite:@"" success:^(NSURL *xmlrpcURL) {
     } failure:^(NSError *error) {
-        [expectation fulfill];
+        NSLog(@"%@", [error localizedDescription]);
+        [expectationEmpty fulfill];
+        errorToCheck = error;
     }];
     [self waitForExpectationsWithTimeout:2 handler:nil];
+    XCTAssertTrue(errorToCheck.domain == WordPressXMLRPCApiErrorDomain, @"Expected to get an WordPressXMLRPCApiErrorDomain error");
+    XCTAssertTrue(errorToCheck.code == WordPressXMLRPCApiEmptyURL, @"Expected to get an WordPressXMLRPCApiEmptyURL error");
+
+    XCTestExpectation *expectationNil = [self expectationWithDescription:@"Call should fail with error when invoking with nil string"];
```

Comment:
Maybe a style thing, but would it be possible to split this test into 3 separate ones to keep them smaller and more focused?